### PR TITLE
Adding PMAT as a dependency, running benchmarks and deploying SME agent

### PR DESCRIPTION
**Pull request by evangriffiths**

### CODE REVIEW

File: tools/prediction_request_embedding/prediction_sentence_embedding.py [hunk 1]
```diff
@@ -1138,7 +1139,7 @@ def run(**kwargs) -> Tuple[str, Optional[str], Optional[Dict[str, Any]], Any]:
         engine = TOOL_TO_ENGINE[tool]
 
         # Extract the event question from the prompt
-        event_question = re.search(r"\"(.+?)\"", prompt).group(1)
+        event_question = prompt
```
Comment (evangriffiths) [inline tools/prediction_request_embedding/prediction_sentence_embedding.py:1142]: Regex search often returns None, causing exception

File: tools/prediction_request_sme/prediction_request_sme.py [hunk 2]
```diff
@@ -303,7 +304,6 @@ def fetch_additional_information(
             json_data["queries"],
             api_key=google_api_key,
             engine=google_engine,
-            num_urls=num_urls,
```
Comment (evangriffiths) [inline tools/prediction_request_sme/prediction_request_sme.py:306]: bugfix

File: tools/prediction_request_sme/prediction_request_sme.py [hunk 3]
```diff
@@ -60,7 +60,8 @@ def count_tokens(text: str, model: str) -> int:
 
 
 NUM_URLS_EXTRACT = 5
-DEFAULT_NUM_WORDS: Dict[str, Optional[int]] = defaultdict(lambda: 300)
+# DEFAULT_NUM_WORDS: Dict[str, Optional[int]] = defaultdict(lambda: 300)
+DEFAULT_NUM_WORDS = 300
```
Comment (evangriffiths) [inline tools/prediction_request_sme/prediction_request_sme.py:64]: bugfix

File: tools/prediction_request_sme/prediction_request_sme.py [hunk 4]
```diff
@@ -202,7 +203,7 @@ def get_urls_from_queries(queries: List[str], api_key: str, engine: str) -> List
             query=query,
             api_key=api_key,
             engine=engine,
-            num=3,  # Number of returned results
+            num=NUM_URLS_EXTRACT,  # Number of returned results
```
Comment (evangriffiths) [inline tools/prediction_request_sme/prediction_request_sme.py:206]: 3 or NUM_URLS_EXTRACT==5? Hmm, let's use global var

**Comment by evangriffiths** [comment]
An example of the research strings returned by the SME mech:
```
- How do NOAA scientists predict the annual global temperature ranking ahead of time? | NOAA Climate.gov
Skip to main content
Menu
Climate.gov
Science & information for a climate-smart nation
Main Menu
Main menu
News & Features
Home
All News & Features
Blogs
ENSO Blog
Polar Vortex Blog
Beyond the Data
- Copernicus: In 2024, the world experienced the warmest January on record | Copernicus
skip to main content
01-access-to-data02-use-cases03-press04-tenders05-contactback-esotc-2020bookcalendarccl-icon-atmosphere-whiteccl-icon-atmosphereccl-icon-climate-whiteccl-icon-climateccl-icon-emergency-whiteccl
- U.S. climate outlook for January 2024 | NOAA Climate.gov
Skip to main content
Menu
Climate.gov
Science & information for a climate-smart nation
Main Menu
Main menu
News & Features
Home
All News & Features
Blogs
ENSO Blog
Polar Vortex Blog
Beyond the Data Blog
Climate And ...
Climate Case Studies
Cli
- Climate Prediction Center - Seasonal Outlook
www.nws.noaa.gov
Home
Site Map
News
Organization
Search
CPC Search
Map Explanations
Official Fcsts
Fcst Tools
About Us
Our Mission
Who We Are
Contact Us
CPC Information
CPC Web Team
Official 90-day Outlooks are issued once each month near mid-month at 8:3
- 2024 could be even hotter than 2023, NASA says
Open in Our App
Get the best experience and stay connected to your community with our Spectrum News app. Learn More
Open in Spectrum News App
Continue in Browser
Toggle navigation
All Boroughs
EDIT
Watch Live
|
Recent Activity
Log Out
Recent Activity
Lo
```

vs. the Embedding mech:
```
- (2024-01-02T11:00:42.724Z, 2024-01-04T20:35:13.630Z):What climate scientists are predicting for the globe in 2024 - The Washington Post. | The planet came closer than ever to that dreaded threshold in recent months, providing a first glimpse of a world where sustained levels of that heat would fuel new weather extremes. | An intense episode of the planet-warming El Niño climate pattern is nearing its peak, and the last time that happened, it pushed the planet to record warmth in 2016.Sign up for the Climate Coach newsletter and get advice for life on our changing planet, in your inbox every Tuesday. | That suggests there will be no imminent slowdown in a surge of global warmth that has supercharged the decades-long trend tied to fossil fuel emissions.

- (2024-01-03T00:00:37+00:00, 2024-01-04T02:18:27+00:00):Increased emissions of greenhouse gases have led to a rapid and steady increase in global temperatures, which in turn is causing catastrophic events all over the world – from Australia and the US experiencing some of the most devastating bushfire seasons ever recorded, locusts swarming across parts of Africa, the Middle East and Asia, decimating crops, and a heatwave in Antarctica that saw temperatures rise above 20C for the first time. | Meanwhile, the Antarctic continent contributes about 1 millimetre per year to sea level rise, which is one-third of the annual global increase. | By 2015, this annual production swelled to 419 million tons and exacerbating plastic waste in the environment. | If we include microplastics into this, the cumulative amount of plastic in the ocean could reach 600 million tons by 2040. | However, even if all greenhouse gas emissions were halted immediately, global temperatures would continue to rise in the coming years. | According to a 2023 study, air pollution in South Asia – one of the most polluted areas in the world – cuts life expectancy by about 5 years.  | Additionally, the last fully intact ice shelf in Canada in the Arctic recently collapsed, having lost about 80 square kilometres – or 40% – of its area over a two-day period in late July, according to the Canadian Ice Service. | Representing arguably the biggest of the environmental problems, this is made all the more concerning considering that last year’s summer triggered the loss of 60 billion tons of ice from Greenland, enough to raise global sea levels by 2.2mm in just two months. | 2023 was the hottest year on record, with global average temperatures at 1.46C above pre-industrial levels and 0.13C higher than the eleven-month average for 2016, currently the warmest calendar year on record. | Economists and environmentalists have urged policymakers for years to increase the price of activities that emit greenhouse gases (one of our biggest environmental problems), the lack of which constitutes the largest market failure, for example through carbon taxes, which will stimulate innovations in low-carbon technologies. | The global demand for fashion and clothing has risen at an unprecedented rate that the fashion industry now accounts for 10% of global carbon emissions, becoming one of the biggest environmental problems of our time. | The smallest change in the pH scale can have a significant impact on the acidity of the ocean. | In July 2022, the WTO banned fishing subsidies to reduce global overfishing in a historic deal. | Studies have shown that the global food system is responsible for up to one-third of all human-caused greenhouse gas emissions, of which 30% comes from livestock and fisheries. | According to satellite data, the Greenland ice sheet lost a record amount of ice in 2019: an average of a million tons per minute throughout the year, one of the biggest environmental problems that has cascading effects. | While the United Nations Fashion Industry Charter for Climate Action sees signatory fashion and textile companies to commit to achieving net zero emission by 2050, a majority of businesses around the world have yet to address their roles in climate change. | According to 2023 data, the continent has lost approximately 7.5 trillion tons of ice since 1997.

- (2024-01-12, unknown):Earth boiled in 2023 — will it happen again in 2024?. | Postdoctoral position in cancer biology is available to carry out projects focused on studying the effects of small molecules in cancer.

- (unknown, unknown):Map of global average surface temperature in 2023 compared to the 1991-2020 average. | Forecast confidence is greatest in New England, where above-average temperatures are most likely given the consistent signals in the various forecasts for above-average temperatures during each week of the month. | To calculate a global average temperature, scientists begin with temperature measurements taken at locations around the globe. | The anticipated cold and wet start to the month in the Pacific Northwest favors an improvement in snowpack and drought, although drought persistence is predicted in the Intermountain West, due to larger precipitation deficits there. | updated in 2024, uses comprehensive data collections of increased global area coverage over both. | Drought is expected to persist in other areas currently experiencing drought, despite the slight tilt toward above-average precipitation in some of these regions, as winter is a drier time of year in these locations. | Why does 1.5C matter and how will future climate change affect the world?The more average temperatures increase, the worse the impacts of climate change become. | Temperatures vary from night to day and between seasonal extremes in the Northern and Southern Hemispheres. | Because their goal is to track changes in temperature, measurements are converted from absolute temperature readings to temperature anomalies—the difference between the observed temperature and the long-term average temperature for each location and date. | The drought outlook for January favors improvement and/or removal in parts of the Pacific Northwest, parts of the Southwest, and in the Southeast extending up the Appalachians to the mid-Atlantic. | Though warming has not been uniform across the planet, the upward trend in the globally averaged temperature shows that more areas are warming than cooling. | In a landmark agreement signed in Paris in 2015, almost 200 countries pledged to try to keep global warming to 1.5C. To achieve this, "net zero" CO2 emissions should be reached by 2050. | What are the effects of climate change so far?The global average temperature increase of 1.2C seen in the past decade might not sound much. | If annual emissions increase more slowly and begin to decline significantly by 2050,  models project temperatures would still be at least 2.4 degrees warmer than the first half of the 20th century, and possibly up to 5.9 degrees warmer. | Yearly surface temperature compared to the 20th-century average from 1880–2023. | Sweden was colder than average for the fourth consecutive month in January and began the month with temperatures that were the coldest observed since 1999. | Probabilities exceed 50% along the Gulf and South Atlantic Coasts, as 1-3 inches of rainfall is predicted during the first week of the month and forecasts at all longer time ranges continue to favor above-average precipitation. | Probabilities for above-average precipitation are also elevated along the West Coast and in the Southwest, but with modest probabilities, due to uncertainty on the location of the primary storm track. | Projected temperature increase relative to the 1901-1960 average depending on which RCP we eventually follow. | Although above-average temperatures persisted in the upper 100 meters of the equatorial Pacific, below-average temperatures were widespread at greater depths | The amount of future warming Earth will experience depends on how much carbon dioxide and other greenhouse gases we emit in coming decades. | In the Southwest, improvement is likely to be more targeted, with a broad area of improvement and removal depicted in areas where there is at least a 40% chance of above-average precipitation during January. | Click and drag slider to compare how drought conditions changed between the end of November and the end of December 2023. | It has now been confirmed that global warming exceeded 1.5C across the 12 month period between February 2023 and January 2024. | The first month of meteorological winter is officially in the books and most folks across the United States are still waiting for winter to arrive. | Although the first week of the month is expected to be dry across the Northern Plains, around the Great Lakes, and in the Ohio Valley, the Week 2 forecast leans wet, so the outlook for these regions was for equal chances of above-, near-, or above-normal precipitation. | In the atmosphere, 500-millibar height pressure anomalies correlate well with temperatures at the Earth's surface. | In addition, it's clear that the rate of warming over the past few decades is much faster than the average rate since the start of the 20th century. | Limiting long-term average temperature rises to 1.5Cis crucial, according to the IPCC.The science is not completely certain, but the consequences of 2C global warming versus 1.5C could include:Extreme hot days would be on average 4C warmer at mid-latitudes (regions outside the poles and tropics), versus 3C at 1.5CSea-level rise would be 0.1m higher than at 1.5C, exposing up to 10 million more people to events including more frequent floodingMore than 99% of coral reefs would be lost, compared with 70-90% at 1.5CTwice the number of plants and vertebrates (animals with a backbone) would be exposed to unsuitable climate conditions across more than half the geographical area where they are foundSeveral hundred million more people may be exposed to climate-related risks and susceptible to poverty by 2050 than at 1.5C.The call to limit temperature rise to 1.5C was partly designed to avoid crossing so-called "tipping points". | Drought removal is predicted in regions currently in moderate drought, with improvement, but not removal, forecast for areas in more significant drought. | Since the MJO is predicted to remain active and slowly progress towards the Maritime Continent and then the western Pacific during the first half of January, potential MJO impacts were also considered. | Things are likely to worsen in the coming decades, but scientists argue urgent action can limit the worst effects of climate change.
```

**Comment by evangriffiths** [comment]
Outputs of `python examples/benchmark_mechs.py`,

benchmark_report.manifold.md:
```
# Comparison Report

## Summary Statistics

| Agents    |   MSE for `p_yes` |   Mean confidence |   % within +-0.05 |   % within +-0.1 |   % within +-0.2 |   % correct outcome |   % precision for `yes` |   % precision for `no` |   % recall for `yes` |   % recall for `no` |   confidence/p_yes error correlation |   Mean info_utility |   Proportion answerable |   Proportion answered | Mean cost ($)   |   Mean time (s) |
|:----------|------------------:|------------------:|------------------:|-----------------:|-----------------:|--------------------:|------------------------:|-----------------------:|---------------------:|--------------------:|-------------------------------------:|--------------------:|------------------------:|----------------------:|:----------------|----------------:|
| random    |         0.129464  |          0.519657 |                16 |               28 |               32 |                  60 |                 40      |                73.3333 |                 50   |             64.7059 |                           -0.0782577 |             nan     |                       1 |                     1 |                 |     7.43675e-05 |
| sme       |         0.0508025 |          0.758    |                24 |               32 |               64 |                  76 |                 61.5385 |               100      |                100   |             70.5882 |                            0.103736  |               0.608 |                       1 |                     1 |                 |    14.4835      |
| embedding |         0.0349453 |          0.832    |                12 |               24 |               72 |                  76 |                 58.3333 |                92.3077 |                 87.5 |             70.5882 |                           -0.318631  |               0.832 |                       1 |                     1 |                 |    27.7407      |

## Markets

| Market Question                                                                                                                                                                           |   random p_yes |   sme p_yes |   embedding p_yes |   reference p_yes |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------:|------------:|------------------:|------------------:|
| [Will a reliable and general household robot be developed before January 1st, 2030?](https://manifold.markets/MatthewBarnett/will-a-reliable-and-general-househo)                         |     0.101319   |        0.6  |               0.6 |         0.33      |
| [Will Biden be the 2024 Democratic Nominee?](https://manifold.markets/NathanpmYoung/will-biden-be-the-2024-democratic-n)                                                                  |     0.459195   |        0.7  |               0.8 |         0.902809  |
| [Will Vladimir Putin be the President of Russia at the end of 2024?](https://manifold.markets/Inosen_Infinity/will-vladimir-putin-be-the-presiden-00a0332740bc)                           |     0.993115   |        0.9  |               0.8 |         0.940065  |
| [Will Joe Biden win the 2024 US Presidential Election?](https://manifold.markets/LarsDoucet/will-joe-biden-win-the-2024-us-pres)                                                          |     0.576438   |        0.55 |               0.4 |         0.478505  |
| [Will there be an AI language model that surpasses ChatGPT and other OpenAI models before the end of 2024?](https://manifold.markets/Gen/will-there-be-an-ai-language-model)              |     0.962034   |        0.5  |               0.6 |         0.377541  |
| [Hamas command center under al-Shifa Hospital?](https://manifold.markets/RichardHanania/hamas-command-center-under-alshifa)                                                               |     0.0609812  |        0.8  |               0.8 |         0.66      |
| [Will the five most valuable members of the S&P 500 be more than 21% of the index's value in 2025?](https://manifold.markets/ByrneHobart/will-the-five-most-valuable-members)             |     0.821543   |        0.8  |               0.6 |         0.775338  |
| [Will China launch a full-scale invasion of Taiwan before 2030?](https://manifold.markets/MetaculusBot/will-china-launch-a-fullscale-invas)                                               |     0.647714   |        0.4  |               0.3 |         0.290895  |
| [Will over 100,000 people be conceived with the help of advanced embryo selection techniques by 2030?](https://manifold.markets/MilfordHammerschmidt/will-over-100000-people-be-conceive) |     0.0775776  |        0.8  |               0.2 |         0.345895  |
| [Will the LK-99 room temp, ambient pressure superconductivity pre-print replicate before 2025?](https://manifold.markets/QuantumObserver/will-the-lk99-room-temp-ambient-pre)             |     0.297979   |        0.3  |               0.2 |         0.0341337 |
| [World gdp growth from 2023 to 2100? (nominal USD, annualized, 10x amplified) (M10,000 subsidy)](https://manifold.markets/JonathanRay/what-will-be-annualized-nominal-wor)                |     0.861852   |        0.7  |               0.5 |         0.561258  |
| [In 2028, will AI be at least as big a political issue as abortion?](https://manifold.markets/ScottAlexander/in-2028-will-ai-be-at-least-as-big)                                          |     0.414746   |        0.3  |               0.3 |         0.401216  |
| [Will Ukraine win the Russo-Ukrainian War?](https://manifold.markets/chrisjbillington/will-ukraine-win-the-russoukrainian)                                                                |     0.996277   |        0.4  |               0.3 |         0.16      |
| [Will OpenAI hint at or claim to have AGI by 2025 end?](https://manifold.markets/firstuserhere/will-openai-hint-at-or-claim-to-hav)                                                       |     0.495169   |        0.4  |               0.2 |         0.197412  |
| [Will the average global temperature in 2024 exceed 2023?](https://manifold.markets/SteveRabin/will-the-average-global-temperature)                                                       |     0.00449264 |        0.6  |               0.7 |         0.556233  |
| [In 2028, will an AI be able to generate a full high-quality movie to a prompt?](https://manifold.markets/ScottAlexander/in-2028-will-an-ai-be-able-to-gener)                             |     0.543527   |        0.2  |               0.6 |         0.310869  |
| [Will a company started by me have a valuation of at least one billion USD by the year 2026?](https://manifold.markets/jim/will-a-company-started-by-me-have-a)                           |     0.444846   |        0.4  |               0.2 |         0.0298051 |
| [Will Binance collapse before the end of 2024?](https://manifold.markets/MarcusAbramovitch/will-binance-collapse-before-the-en)                                                           |     0.570812   |        0.75 |               0.2 |         0.0928331 |
| [Will Bougainville be a country by end of 2027? [Showcase: M$10,000 subsidy]](https://manifold.markets/Conflux/will-bougainville-be-a-country-by-e)                                       |     0.729826   |        0.6  |               0.6 |         0.699596  |
| [By 2050, will there be a cure to aging?](https://manifold.markets/DanielKilian/by-2050-will-there-be-a-cure-to-agi)                                                                      |     0.165346   |        0.2  |               0.2 |         0.21      |
| [Will AI be a major topic during the 2024 presidential debates in the United States? (see criteria)](https://manifold.markets/MatthewBarnett/will-ai-be-a-major-topic-during-the)         |     0.0367835  |        0.6  |               0.6 |         0.33      |
| [By 2030, will there be a cure to aging?](https://manifold.markets/DanielKilian/by-2030-will-there-be-a-cure-to-agi)                                                                      |     0.172773   |        0.1  |               0.4 |         0.097841  |
| [Will a room-temperature, atmospheric pressure superconductor be discovered before 2030?](https://manifold.markets/AaronLehmann/will-a-roomtemperature-atmospheric)                       |     0.41708    |        0.2  |               0.2 |         0.0714326 |
| [Will we have end to end AI generated anime series by the end of 2024?](https://manifold.markets/firstuserhere/will-we-have-end-to-end-ai-generate-5927e7f2d735)                          |     0.102943   |        0.3  |               0.7 |         0.25      |
| [Someone dies wearing Apple Vision Pro](https://manifold.markets/GabeGarboden/someone-dies-wearing-apple-vision-p)                                                                        |     0.449235   |        0.7  |               0.6 |         0.81      |

## Expected value

| Agent     |   Mean expected returns |   Median expected returns |   Total expected returns |
|:----------|------------------------:|--------------------------:|-------------------------:|
| random    |                 15.681  |                   30.8209 |                  392.024 |
| sme       |                 36.4111 |                   41.821  |                  910.277 |
| embedding |                 35.7698 |                   41.821  |                  894.245 |

| Market Question                                                                                           |    random |       sme |   embedding |
|:----------------------------------------------------------------------------------------------------------|----------:|----------:|------------:|
| Will a reliable and general household robot be developed before January 1st, 2030?                        |  34       | -34       |   -34       |
| Will Biden be the 2024 Democratic Nominee?                                                                | -80.5617  |  80.5617  |    80.5617  |
| Will Vladimir Putin be the President of Russia at the end of 2024?                                        |  88.013   |  88.013   |    88.013   |
| Will Joe Biden win the 2024 US Presidential Election?                                                     |  -4.29907 |  -4.29907 |     4.29907 |
| Will there be an AI language model that surpasses ChatGPT and other OpenAI models before the end of 2024? | -24.4919  |  24.4919  |   -24.4919  |
| Hamas command center under al-Shifa Hospital?                                                             | -32       |  32       |    32       |
| Will the five most valuable members of the S&P 500 be more than 21% of the index's value in 2025?         |  55.0676  |  55.0676  |    55.0676  |
| Will China launch a full-scale invasion of Taiwan before 2030?                                            | -41.821   |  41.821   |    41.821   |
| Will over 100,000 people be conceived with the help of advanced embryo selection techniques by 2030?      |  30.8209  | -30.8209  |    30.8209  |
| Will the LK-99 room temp, ambient pressure superconductivity pre-print replicate before 2025?             |  93.1733  |  93.1733  |    93.1733  |
| World gdp growth from 2023 to 2100? (nominal USD, annualized, 10x amplified) (M10,000 subsidy)            |  12.2516  |  12.2516  |   -12.2516  |
| In 2028, will AI be at least as big a political issue as abortion?                                        |  19.7567  |  19.7567  |    19.7567  |
| Will Ukraine win the Russo-Ukrainian War?                                                                 | -68       |  68       |    68       |
| Will OpenAI hint at or claim to have AGI by 2025 end?                                                     |  60.5177  |  60.5177  |    60.5177  |
| Will the average global temperature in 2024 exceed 2023?                                                  | -11.2467  |  11.2467  |    11.2467  |
| In 2028, will an AI be able to generate a full high-quality movie to a prompt?                            | -37.8261  |  37.8261  |   -37.8261  |
| Will a company started by me have a valuation of at least one billion USD by the year 2026?               |  94.039   |  94.039   |    94.039   |
| Will Binance collapse before the end of 2024?                                                             | -81.4334  | -81.4334  |    81.4334  |
| Will Bougainville be a country by end of 2027? [Showcase: M$10,000 subsidy]                               |  39.9191  |  39.9191  |    39.9191  |
| By 2050, will there be a cure to aging?                                                                   |  58       |  58       |    58       |
| Will AI be a major topic during the 2024 presidential debates in the United States? (see criteria)        |  34       | -34       |   -34       |
| By 2030, will there be a cure to aging?                                                                   |  80.4318  |  80.4318  |    80.4318  |
| Will a room-temperature, atmospheric pressure superconductor be discovered before 2030?                   |  85.7135  |  85.7135  |    85.7135  |
| Will we have end to end AI generated anime series by the end of 2024?                                     |  50       |  50       |   -50       |
| Someone dies wearing Apple Vision Pro                                                                     | -62       |  62       |    62       |
```

benchmark_report.polymarket.md:
```
# Comparison Report

## Summary Statistics

| Agents    |   MSE for `p_yes` |   Mean confidence |   % within +-0.05 |   % within +-0.1 |   % within +-0.2 |   % correct outcome |   % precision for `yes` |   % precision for `no` |   % recall for `yes` |   % recall for `no` |   confidence/p_yes error correlation |   Mean info_utility |   Proportion answerable |   Proportion answered | Mean cost ($)   |   Mean time (s) |
|:----------|------------------:|------------------:|------------------:|-----------------:|-----------------:|--------------------:|------------------------:|-----------------------:|---------------------:|--------------------:|-------------------------------------:|--------------------:|------------------------:|----------------------:|:----------------|----------------:|
| random    |          0.194716 |          0.497569 |          10       |          13.3333 |          26.6667 |             63.3333 |                 18.1818 |                89.4737 |                   50 |             65.3846 |                             0.028596 |          nan        |                       1 |                     1 |                 |     6.35783e-06 |
| sme       |          0.179844 |          0.735    |           6.66667 |          10      |          36.6667 |             43.3333 |                 11.7647 |                84.6154 |                   50 |             42.3077 |                             0.670547 |            0.613333 |                       1 |                     1 |                 |    13.3812      |
| embedding |          0.127934 |          0.846667 |          10       |          16.6667 |          46.6667 |             56.6667 |                 10      |                85      |                   25 |             65.3846 |                            -0.190366 |            0.84     |                       1 |                     1 |                 |    18.2263      |

## Markets

| Market Question                                                                                                                                                                                                        |   random p_yes |   sme p_yes |   embedding p_yes |   reference p_yes |
|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------:|------------:|------------------:|------------------:|
| [Will The Holdovers win the Oscar for Best Original Screenplay?](https://polymarket.com/event/will-the-holdovers-win-the-oscar-for-best-original-screenplay)                                                           |      0.258399  |        0.3  |               0.2 |            0.365  |
| [Will "American Fiction" win the Oscar for Best Picture?](https://polymarket.com/event/will-american-fiction-win-the-oscar-for-best-picture)                                                                           |      0.357301  |        0.7  |               0.8 |            0.0105 |
| [Will the median gas price per blob be between 0.01 and 0.1 ETH one month after EIP-4844?](https://polymarket.com/event/will-the-median-gas-price-per-blob-be-between-0pt01-and-0pt1-eth-one-month-after-eip-4844)     |      0.214483  |        0.7  |               0.7 |            0.095  |
| [OpenSea token in 2024?](https://polymarket.com/event/opensea-token-in-2024)                                                                                                                                           |      0.182842  |        0.6  |               0.5 |            0.225  |
| [Will Warpcast flip X on the app store in February?](https://polymarket.com/event/will-warpcast-flip-x-on-the-app-store-in-february)                                                                                   |      0.164934  |        0.6  |               0.2 |            0      |
| [Will Emma Stone win the Oscar for Best Actress?](https://polymarket.com/event/will-emma-stone-win-the-oscar-for-best-actress)                                                                                         |      0.534727  |        0.4  |               0.6 |            0.595  |
| [Will the median gas price per blob be between 0.001 and 0.01 ETH one month after EIP-4844?](https://polymarket.com/event/will-the-median-gas-price-per-blob-be-between-0pt001-and-0pt01-eth-one-month-after-eip-4844) |      0.471505  |        0.8  |               0.2 |            0.24   |
| [Will US gain 250k-300k jobs in February?](https://polymarket.com/event/will-us-gain-250k-300k-jobs-in-february)                                                                                                       |      0.440074  |        0.7  |               0.6 |            0.155  |
| [Will Maestro win the Oscar for Best Original Screenplay?](https://polymarket.com/event/will-maestro-win-the-oscar-for-best-original-screenplay)                                                                       |      0.781414  |        0.35 |               0.4 |            0.007  |
| [Will May December win the Oscar for Best Original Screenplay?](https://polymarket.com/event/will-may-december-win-the-oscar-for-best-original-screenplay)                                                             |      0.0227998 |        0.2  |               0.2 |            0.007  |
| [Fed rate cut by June 12?](https://polymarket.com/event/fed-rate-cut-by-june-12)                                                                                                                                       |      0.506591  |        0.2  |               0.4 |            0.677  |
| [Will Ryan Gosling win the Oscar for Best Supporting Actor?](https://polymarket.com/event/will-ryan-gosling-win-the-oscar-for-best-supporting-actor)                                                                   |      0.467964  |        0.2  |               0.6 |            0.07   |
| [Tiktok banned in the US by June 30?](https://polymarket.com/event/tiktok-banned-in-the-us-by-june-30)                                                                                                                 |      0.478953  |        0.7  |               0.2 |            0.0545 |
| [Will Poor Things win the Oscar for Best Adapted Screenplay?](https://polymarket.com/event/will-poor-things-win-the-oscar-for-best-adapted-screenplay)                                                                 |      0.571584  |        0.2  |               0.4 |            0.155  |
| [Ethereum spot ETF approved by Mar 31?](https://polymarket.com/event/ethereum-spot-etf-approved-by-mar-31)                                                                                                             |      0.480003  |        0.2  |               0.2 |            0.042  |
| [Will Joe Biden be President of the United States on April 1, 2024?](https://polymarket.com/event/will-joe-biden-be-president-of-the-united-states-on-april-1-2024)                                                    |      0.304724  |        0.6  |               0.2 |            0.9665 |
| [Haniyeh still in Hamas leadership through March 1?](https://polymarket.com/event/haniyeh-still-in-hamas-leadership-through-march-1)                                                                                   |      0.0643164 |        0.8  |               0.2 |            0.525  |
| [[Single Market] Will Vivek Ramaswamy win the U.S. 2024 Republican presidential nomination?](https://polymarket.com/event/will-vivek-ramaswamy-win-the-us-2024-republican-presidential-nomination)                     |      0.277437  |        0.2  |               0.3 |            0.061  |
| [Will Donald Trump win the popular vote in the 2024 Presidential Election?](https://polymarket.com/event/will-donald-trump-win-the-popular-vote-in-the-2024-presidential-election)                                     |      0.70024   |        0.45 |               0.4 |            0.305  |
| [2024 presidential election: Democrats win by 35-64](https://polymarket.com/event/2024-presidential-election-democrats-win-by-35-64)                                                                                   |      0.588086  |        0.6  |               0.6 |            0.105  |
| [EIP-4844 doesn't launch by end of 2024?](https://polymarket.com/event/eip-4844-doesnt-launch-by-end-of-2024)                                                                                                          |      0.764468  |        0.6  |               0.2 |            0.0125 |
| [Will 'Dune: Part Two' gross between $70m-80m opening weekend?](https://polymarket.com/event/will-dune-part-two-gross-between-70m-80m-opening-weekend)                                                                 |      0.101068  |        0.6  |               0.6 |            0.365  |
| [Will Yorgos Lanthimos win the Oscar for Best Director?](https://polymarket.com/event/will-yorgos-lanthimos-win-the-oscar-for-best-director)                                                                           |      0.481928  |        0.2  |               0.2 |            0.0315 |
| [Will Mark Ruffalo win the Oscar for Best Supporting Actor?](https://polymarket.com/event/will-mark-ruffalo-win-the-oscar-for-best-supporting-actor)                                                                   |      0.874278  |        0.2  |               0.6 |            0.02   |
| [SBF sentenced to 50+ years?](https://polymarket.com/event/sbf-sentenced-to-50-years)                                                                                                                                  |      0.464903  |        0.7  |               0.2 |            0.235  |
| [Will Ron DeSantis win the 2024 Republican Presidential Nomination?](https://polymarket.com/event/will-ron-desantis-win-the-2024-republican-presidential-nomination)                                                   |      0.492242  |        0.6  |               0.2 |            0.006  |
| [Will Trump tweet again by April 1?](https://polymarket.com/event/will-trump-tweet-again-by-april-1)                                                                                                                   |      0.157241  |        0.85 |               0.6 |            0.195  |
| [Will 'Deadpool 3' gross most in 2024?](https://polymarket.com/event/will-deadpool-3-gross-most-in-2024)                                                                                                               |      0.757118  |        0.6  |               0.6 |            0.205  |
| [Will Sarah Huckabee Sanders win the 2024 Republican VP nomination?](https://polymarket.com/event/will-sarah-huckabee-sanders-win-the-2024-republican-vp-nomination)                                                   |      0.956662  |        0.7  |               0.2 |            0.0325 |
| [Will Lily Gladstone win the Oscar for Best Actress?](https://polymarket.com/event/will-lily-gladstone-win-the-oscar-for-best-actress)                                                                                 |      0.636895  |        0.3  |               0.4 |            0.335  |

## Expected value

| Agent     |   Mean expected returns |   Median expected returns |   Total expected returns |
|:----------|------------------------:|--------------------------:|-------------------------:|
| random    |                15.8621  |                      35.4 |                    460   |
| sme       |                -5.12414 |                     -27   |                   -148.6 |
| embedding |                14.0207  |                      33   |                    406.6 |

| Market Question                                                                            |   random |   sme |   embedding |
|:-------------------------------------------------------------------------------------------|---------:|------:|------------:|
| Will The Holdovers win the Oscar for Best Original Screenplay?                             |     27   |  27   |        27   |
| Will "American Fiction" win the Oscar for Best Picture?                                    |     97.9 | -97.9 |       -97.9 |
| Will the median gas price per blob be between 0.01 and 0.1 ETH one month after EIP-4844?   |     81   | -81   |       -81   |
| OpenSea token in 2024?                                                                     |     55   | -55   |        55   |
| Will Warpcast flip X on the app store in February?                                         |    nan   | nan   |       nan   |
| Will Emma Stone win the Oscar for Best Actress?                                            |     19   | -19   |        19   |
| Will the median gas price per blob be between 0.001 and 0.01 ETH one month after EIP-4844? |     52   | -52   |        52   |
| Will US gain 250k-300k jobs in February?                                                   |     69   | -69   |       -69   |
| Will Maestro win the Oscar for Best Original Screenplay?                                   |    -98.6 |  98.6 |        98.6 |
| Will May December win the Oscar for Best Original Screenplay?                              |     98.6 |  98.6 |        98.6 |
| Fed rate cut by June 12?                                                                   |     35.4 | -35.4 |       -35.4 |
| Will Ryan Gosling win the Oscar for Best Supporting Actor?                                 |     86   |  86   |       -86   |
| Tiktok banned in the US by June 30?                                                        |     89.1 | -89.1 |        89.1 |
| Will Poor Things win the Oscar for Best Adapted Screenplay?                                |    -69   |  69   |        69   |
| Ethereum spot ETF approved by Mar 31?                                                      |     91.6 |  91.6 |        91.6 |
| Will Joe Biden be President of the United States on April 1, 2024?                         |    -93.3 |  93.3 |       -93.3 |
| Haniyeh still in Hamas leadership through March 1?                                         |     -5   |   5   |        -5   |
| [Single Market] Will Vivek Ramaswamy win the U.S. 2024 Republican presidential nomination? |     87.8 |  87.8 |        87.8 |
| Will Donald Trump win the popular vote in the 2024 Presidential Election?                  |    -39   |  39   |        39   |
| 2024 presidential election: Democrats win by 35-64                                         |    -79   | -79   |       -79   |
| EIP-4844 doesn't launch by end of 2024?                                                    |    -97.5 | -97.5 |        97.5 |
| Will 'Dune: Part Two' gross between $70m-80m opening weekend?                              |     27   | -27   |       -27   |
| Will Yorgos Lanthimos win the Oscar for Best Director?                                     |     93.7 |  93.7 |        93.7 |
| Will Mark Ruffalo win the Oscar for Best Supporting Actor?                                 |    -96   |  96   |       -96   |
| SBF sentenced to 50+ years?                                                                |     53   | -53   |        53   |
| Will Ron DeSantis win the 2024 Republican Presidential Nomination?                         |     98.8 | -98.8 |        98.8 |
| Will Trump tweet again by April 1?                                                         |     61   | -61   |       -61   |
| Will 'Deadpool 3' gross most in 2024?                                                      |    -59   | -59   |       -59   |
| Will Sarah Huckabee Sanders win the 2024 Republican VP nomination?                         |    -93.5 | -93.5 |        93.5 |
| Will Lily Gladstone win the Oscar for Best Actress?                                        |    -33   |  33   |        33   |
```

**Review by kongzii — APPROVED** [review]

**Pull request closed without merging**